### DISCUSSION TITLE
‹kfir/blog› | FizzBuzz: The Rusty Way

### DISCUSSION THREAD

**Discussion by giscus** [kfirfitousi/blog#53] [Comments]
# ‹kfir/blog› | FizzBuzz: The Rusty Way

Implementing FizzBuzz in Rust, and learning about enums and traits along the way.

https://blog.kfirfitousi.com/posts/rust/fizzbuzz

**Comment by tykak** [comment]
ПРИДУРОК

**Reply by bcastillo-2022474** [reply]
asdasdasd

**Reply by bcastillo-2022474** [reply]
asdasdasd

**Comment by bcastillo-2022474** [comment]
hola
